### ISSUE TITLE
add option to unset all other keys before "set"ing

### ISSUE THREAD

**Issue by TomKaltz** [entur/firebase-config-sync#3]
Not sure if this should be handled by the script but it would be a nice option

**Comment by draperunner** [comment]
I agree, it would be a nice feature. We could call it `--force`, `--clear`, `--purge` or something like that.

I don't think Firebase has a "clear all" feature, so we might have to download the existing keys, figure out which of them should be deleted, delete them one-by-one, then set the keys that should be set.

**Comment by draperunner** [comment]
A `purge` command has been implemented now, @TomKaltz. Does this satisfy your needs?

https://github.com/entur/firebase-config-sync#purge
